### DISCUSSION TITLE
what is the best way to extend components

### DISCUSSION THREAD

**Discussion by augustobr1to** [zendeskgarden/react-components#1360] [Q&A] [answered]
I want to create new components and extend some components like DatePicker. What is the best way to achieve this? Fork? Use yarn new command?

**Comment by jzempel** [comment, marked as answer]
Following from the previous [comment](https://github.com/zendeskgarden/react-components/discussions/1345#discussioncomment-2595628), I think your best bet will be to fork the repo. Garden is not positioned to receive net new components via external contribution.

Please consider sharing your extensions back here. We would love to see what you're able to do with Garden. 🙏

**Reply by augustobr1to** [reply]
Thank you @jzempel we're extending some garden components and will be a pleasure to share it for community!
